scripts/ninja.js

```diff
@@ -1357,73 +1352,42 @@ include body.ninja
 exports.updateDev = updateDev;
 exports.updateRelease = updateRelease;
 
-/**
- *
- * @param {string} dir
- */
-function readdirSync(dir) {
-  return fs.readdirSync(dir, "ascii");
-}
-/**
- * @type {string[]}
- */
-var black_list = [];
-/**
- *
- * @param {string} dir
- */
-function test(dir) {
-  return readdirSync(path.join(jscompDir, dir))
-    .filter(x => {
-      return (
-        (x.endsWith(".ml") || x.endsWith(".mli")) &&
-        !(x.endsWith(".cppo.ml") || x.endsWith(".cppo.mli")) &&
-        !(x.endsWith(".pp.ml") || x.endsWith(".pp.mli")) &&
-        !black_list.some(name => x.includes(name))
-      );
-    })
-    .map(x => path.join(dir, x));
-}
-
-/**
- * Built cppo.exe for dev purpose
- */
 function preprocessorNinjaSync() {
   var cppoNative = `
-${cppoRule()}
+${cppoRule("-n")}
```

Comment:
What does this do?

Reply:
Prevent generating lines like
```ocaml
# 1 "others/hashmap.cppo.res"
```
which the ReScript parser does not like.